marti_common_msgs/msg/HealthStatus.msg

```diff
@@ -0,0 +1,12 @@
+# This message is intended for use as a light-weight machine status indicator for robot consumption
+# whereas Diagnostics are for human consumption
+
+int8 OK=0    # Initialized with no issues
```

Comment:
Still 'listening in' and happened to see this comment: `byte` is actually deprecated. See [wiki/msg - Message Description Specification - Field Types](http://wiki.ros.org/msg#Message_Description_Specification).